vagrant/Vagrantfile

```diff
@@ -34,10 +34,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-tftp/node_modules/on-core/" 
             target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/" 
           end
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/RackHD/packer/ansible/roles/monorail/files/auth_config", "/home/vagrant/opt/monorail/"
```

Comment:
@jlongever OK, but the CONFIG_DIR was commented out by default. Do I need to update in somewhere else to set the CONFIG_DIR ?